dcicutils/ff_utils.py

```diff
@@ -1010,21 +1010,51 @@ def remove_keys(my_dict, remove_list):
     return store, list(item_uuids)
 
 
-def get_health_page(key=None, ff_env=None):
-    """
-    Simple function to return the json for a FF health page given keys or
-    ff_env. Will return json containing an error rather than raising an
-    exception if this fails, since this function should tolerate failure
-    """
+def _get_page(page='/health', key=None, ff_env=None):
+    """ Wrapper for commonly used code to GET a page from an environment
+        Given keys or ff_env, will return json containing an error rather than raising an
+        exception if this fails, since this function should tolerate failure """
     try:
         auth = get_authentication_with_server(key, ff_env)
-        health_res = authorized_request(auth['server'] + '/health', auth=auth, verb='GET')
+        health_res = authorized_request(auth['server'] + page, auth=auth, verb='GET')
         ret = get_response_json(health_res)
     except Exception as exc:
         ret = {'error': str(exc)}
     return ret
 
 
+def get_health_page(key=None, ff_env=None):
+    """
+    Simple function to return the json for a FF health page
+    """
+    return _get_page(page='/health', key=key, ff_env=ff_env)
+
+
+def get_counts_page(key=None, ff_env=None):
+    """ Gets DB/ES counts page in JSON """
+    return _get_page(page='/counts', key=key, ff_env=ff_env)
+
+
+def get_indexing_status(key=None, ff_env=None):
+    """ Gets indexing status counts page in JSON """
+    return _get_page(page='/indexing_status', key=key, ff_env=ff_env)
+
+
+def are_counts_even(env):
```

Comment:
This will always return a truthy value so should not have a name that looks like a predicate. Maybe call it `check_counts_even` or `counts_summary` or something like that.

Reply:
This might be a place where returning a `namedtuple` would be a good idea so the person receiving it can access the parts by name instead of by position if they want only one, e.g., if the parts were `are_even` and `summary_total` then `check_counts(...).are_even` serves as a predicate.